pypeit/sensfunc.py

```diff
@@ -193,7 +218,9 @@ def __init__(self, spec1dfile, sensfile, par=None, debug=False):
         self.spectrograph.dispname = header['DISPNAME']
 
         # Get the algorithm parameters
-        self.par = self.spectrograph.default_pypeit_par()['sensfunc'] if par is None else par
+        #self.par = self.spectrograph.default_pypeit_par()['sensfunc'] if par is None else par
```

Comment:
I don't think this should be commented out -- we need `self.par`, _e.g._, in line 240 below:
```
self.splice_multi_det = True if self.par['multi_spec_det'] is not None else False
```

Reply:
Good catch.